thirdparty/icu4c/godot_data.json

```diff
@@ -6,5 +6,6 @@
         brkitr_tree: include
         misc: include
         normalization: include
+        confusables: include
     }
 }
```

Comment:
Not new in this PR but seems to be newline at EOF. That file is probably skipped by our format checks due to being in `thirdparty`, but since it's Godot-specific, might be worth fixing anyway.